tools/gen_addons_table.py

```diff
@@ -84,19 +86,24 @@ def replace_in_readme(readme_path, header, rows_available, rows_unported):
         f.write(readme)
 
 
-@click.command()
+@click.command(help=__doc__)
 @click.option('--commit/--no-commit',
               help="git commit changes to README.rst, if any.")
-def gen_addons_table(commit):
-    readme_path = 'README.md'
+@click.option('--readme-path', default="README.md",
+              type=click.Path(dir_okay=False, file_okay=True, exists=True),
+              help="README.md file with addon table markers")
+@click.option('--addons-dir', default="",
```

Comment:
```suggestion
@click.option('--addons-dir', default=".",
```

because `os.listdir("")` fails